grakn-core/src/main/java/ai/grakn/graql/admin/Answer.java

```diff
@@ -67,7 +66,7 @@
     Concept get(Var var);
 
     @CheckReturnValue
-    ImmutableMap<Var, Concept> map();
```

Comment:
why?

Reply:
renamed immutableMap() to just map(). "immutable" prefix is implementation detail, like HashMap, or HashSet - we always call for "Map".